test/remote/manifests/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- e2e-repositories.yaml
+
+images:
+- name: quay.io/jfischer-redhat/argocd-e2e-cluster
```

Comment:
image name is supposed to be overridden right? In this case let avoid using personal repo name here. May be rename to just `argocd-e2e-cluster` ?

Reply:
Yes, good hint. I have changed it to be just `argocd-e2e-cluster`. Thanks!